.dev/docker/windows/Dockerfile

```diff
@@ -9,40 +9,43 @@ ARG platform
 ARG CHANNEL_URL=https://aka.ms/vs/16/release/channel
 ADD ${CHANNEL_URL} C:\TEMP\VisualStudio.chman
 
-# Download and install Build Tools for Visual Studio 2019 for native desktop workload.
-ADD https://aka.ms/vs/16/release/vs_buildtools.exe C:\TEMP\vs_buildtools.exe
-
-RUN C:\TEMP\vs_buildtools.exe --quiet --wait --norestart --nocache `
-    --channelUri C:\TEMP\VisualStudio.chman `
-    --installChannelUri C:\TEMP\VisualStudio.chman `
-    --add Microsoft.VisualStudio.Workload.VCTools --includeRecommended`
-    --installPath C:\BuildTools
-
 SHELL ["powershell", "-Command", "$ErrorActionPreference = 'Stop'; $ProgressPreference = 'SilentlyContinue';"]
 
-RUN iex ((New-Object System.Net.WebClient).DownloadString('https://chocolatey.org/install.ps1'))
-RUN choco install cmake --installargs 'ADD_CMAKE_TO_PATH=System' -y --no-progress
+# Download and install Build Tools for Visual Studio 2019 for native desktop
+RUN wget 'https://aka.ms/vs/16/release/vs_buildtools.exe' -OutFile 'C:\TEMP\vs_buildtools.exe'; `
+    Start-Process -FilePath C:\TEMP\vs_buildtools.exe -ArgumentList `
+    "--quiet",                                                      `
+    "--norestart",                                                  `
+    "--nocache",                                                    `
+    "--installPath",                                                `
+    "C:\BuildTools",                                                `
+    "--wait",                                                       `
+    "--channelUri",                                                 `
+    "C:\TEMP\VisualStudio.chman",                                   `
+    "--installChannelUri",                                          `
+    "C:\TEMP\VisualStudio.chman",                                   `
+    "--add",                                                        `
+    "Microsoft.VisualStudio.Workload.VCTools",                      `
+    "--includeRecommended"                                          `
+    -Wait -PassThru;                                                `
+    del c:\temp\vs_buildtools.exe;                                  
+    
+RUN iex ((New-Object System.Net.WebClient).DownloadString('https://chocolatey.org/install.ps1'));        `
+    choco install cmake --installargs 'ADD_CMAKE_TO_PATH=System' -y --no-progress
 
 # Install vcpkg
-ADD https://github.com/Microsoft/vcpkg/archive/master.zip C:\TEMP\vcpkg-master.zip
-
 RUN $ErrorActionPreference = 'Stop';                                  `
+    wget 'https://github.com/Microsoft/vcpkg/archive/master.zip' -OutFile 'C:\TEMP\vcpkg-master.zip'; `
```

Comment:
Why not use git instead? Isn't git anyways required to fetch the PCL repo (and by vcpkg)?

Reply:
I'd have to wget git instead :) and install it... 
The PCL repo is fetched by the azure pipline framework and independent of the container.
VCPKG install the portable git and probably is a bit picky about which version, but I think this is the easiest way.

Reply:
> a bit picky about which version

I didn't get this, but let's do this your way instead (it's also not much of an issue)